src/task-manager.ts

```diff
@@ -68,4 +79,17 @@ export class TaskManager implements ValidationTaskManager {
   createGetItemValidationReviewStatusesTask(member: Member): GetItemValidationReviewStatusesTask {
     return new GetItemValidationReviewStatusesTask(member, this.itemValidationService);
   }
+  createGetItemValidationGroupsTask(member: Member, itemValidationId: string): GetItemValidationGroupsTask {
+    return new GetItemValidationGroupsTask(member, this.itemValidationService, { itemValidationId });
+  }
+  createToggleEnabledForItemValidationProcessTask(
+    member: Member,
+    id: string,
+    data: { enabled: boolean },
```

Comment:
You should define the type `ToggleEnabledForItemValidationProcessTask` and reuse it. I would usually go for `ToggleEnabledForItemValidationProcessTaskInput`